contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -5393,16 +5393,16 @@ makeInsertBulkStatement(TSqlParser::Dml_statementContext *ctx)
 				else if (pg_strcasecmp("KEEP_NULLS", ::getFullText(option_list[i]->id()).c_str()) == 0)
 					stmt->keep_nulls = true;
 
-				else if (pg_strcasecmp("CHECK_CONSTRAINTS", ::getFullText(option_list[i]->id()).c_str()) == 0)
+				else if (escape_hatch_insert_bulk_options == EH_STRICT && pg_strcasecmp("CHECK_CONSTRAINTS", ::getFullText(option_list[i]->id()).c_str()) == 0)
 					throw PGErrorWrapperException(ERROR, ERRCODE_FEATURE_NOT_SUPPORTED, "insert bulk option check_constraints is not yet supported in babelfish", getLineAndPos(bulk_ctx->WITH()));
 
-				else if (pg_strcasecmp("FIRE_TRIGGERS", ::getFullText(option_list[i]->id()).c_str()) == 0)
+				else if (escape_hatch_insert_bulk_options == EH_STRICT && pg_strcasecmp("FIRE_TRIGGERS", ::getFullText(option_list[i]->id()).c_str()) == 0)
 					throw PGErrorWrapperException(ERROR, ERRCODE_FEATURE_NOT_SUPPORTED, "insert bulk option fire_triggers is not yet supported in babelfish", getLineAndPos(bulk_ctx->WITH()));
 
-				else if (pg_strcasecmp("TABLOCK", ::getFullText(option_list[i]->id()).c_str()) == 0)
+				else if (escape_hatch_insert_bulk_options == EH_STRICT && pg_strcasecmp("TABLOCK", ::getFullText(option_list[i]->id()).c_str()) == 0)
 					throw PGErrorWrapperException(ERROR, ERRCODE_FEATURE_NOT_SUPPORTED, "insert bulk option tablock is not yet supported in babelfish", getLineAndPos(bulk_ctx->WITH()));
 
-				else
+				else if (escape_hatch_insert_bulk_options == EH_STRICT)
```

Comment:
1. This should not be under escape_hatch because it is not a unsupported feature
2. Do we have a test case verifying the error message "invalid insert bulk option"?

Reply:
1. I have reverted escape_hatch after talking to Alex. We are now ignoring the error message
2. Added